web/implementation-reports/svnmentions.md

```diff
@@ -0,0 +1,109 @@
+# SVNmentions
+
+Implementation Home Page URL: https://github.com/carrvo/SVNmentions
+Version: v1.0
+
+Source Code repo URL(s) (optional): 
+* [ ] 100% open source implementation
+
+Programming Language(s): PHP
+
+Developer(s): carrvo
+
+Implementation Classes (Sender and/or Receiver): Receiver
+
+
+## Receiving
+
+Indicate which type of response the receiver provides:
+
+* [x] HTTP 200 - Receiver synchronously processes the Webmention request (not recommended)
+* [ ] HTTP 201 - Receiver creates a status URL the sender can use to check the status of the Webmention
+* [ ] HTTP 202 - Receiver processes the Webmention asynchronously
+
+Describe the response body (if any) which is returned in the request:
+
+
+### Request Verification (3.2.1)
+
+* [x] Verifies source and target are valid URLs, rejecting with HTTP 400 (MUST)
+* [x] Verifies that target is a valid resource for which the receiver accepts Webmentions, rejecting with HTTP 400 (SHOULD)
+* [x] Ignores fragment when checking if target is supported (SHOULD)
+
+### Webmention Verification (3.2.2)
+
+* [ ] Verification is processed asynchronously (SHOULD)
+* [x] Follows at least one HTTP redirect on source URL (MUST)
+* [x] Respects a self-imposed limit on number of HTTP redirects to follow (MUST)
+
+#### Source URL content-types supported
+
+Please list the content types that your implementation supports when checking if the source document links to the target URL.
+
+* [x] HTML
+* [ ] Other: ______
+
+
+### HTML Verification (3.2.2)
+
+The tests below apply when the source document is HTML.
+
+* [x] Accepts a Webmention where the target URL is in an `<a>` tag
+* [x] Accepts a Webmention where the target URL is in an `<img>` tag
+* [x] Accepts a Webmention where the target URL is in an `<video>` tag
+* [x] Accepts a Webmention where the target URL is in an `<audio>` tag
+* [ ] Rejects a Webmention where the target URL is in the document as text
+* [ ] Rejects a Webmention where the target URL is in an `<a>` tag **inside an HTML comment**
+* [x] Rejects a Webmention where the target URL is not in the document
+
+
+### Webmention Display/Use
+
+* [ ] The receiver displays data from the source URL on the target post (MAY)
+
+* [x] The receiver recognizes that the source URL is a "comment" or "reply" to the post
+ * [ ] using HTML markup: __________
```

Comment:
Assumes all are comments.
What does `using HTML markup` refer to? The markup that the receiver uses on the target? Or the markup that it searches for on the source?